core/src/epicli/data/common/ansible/playbooks/helm.yml

```diff
@@ -1,7 +1,7 @@
 ---
-- hosts: helm
-  become: true
-  become_method: sudo
-  become_flags: --login
+- hosts: 127.0.0.1
+  gather_facts: false
+  connection: local
+  become: false
   roles:
     - helm
```

Comment:
Please note there is no way to disable helm after that change. Is this intended?

Reply:
nope, fixed